llm/ext_server/server.cpp

```diff
@@ -913,17 +878,9 @@ struct llama_server_context
         slot.sampled = result.tok;
 
         // search stop word and delete it
-        if (!llama_token_is_eog(model, result.tok))
-            slot.generated_text += token_str;
-
+        slot.generated_text += token_str;
```

Comment:
This (and further down) was actually a bug fix that we applied locally - check out 93ac3760cb4abdd0f54ad755ed15ec4254026282

Reply:
Thanks for finding this! I definitely missed that this was an intentional diff. Your earlier commit `cherry-pick`'ed nicely.